### PR TITLE
replace back button with link - 211

### LINKED ISSUE #211: Config Viewer - Breadcrumbs
Replace the "back" button with a link in the top nav bar back to home

### PR DESCRIPTION
**Pull request by gyanpgupta**
Closes #211

### CODE REVIEW

File: src/pages/config/config-details.js [hunk 1]
```diff
@@ -91,16 +91,13 @@ export function ConfigDetailsPage() {
           ? `Config Details for ${configDetails.name}`
           : "Config Details"
       }
+      button={<Link className="underline mr-3"
+        to={`/topology`}>
```
Comment (moshloop) [inline src/pages/config/config-details.js:95]: @gyanpgupta why the change from /config to /topology ?

Comment (gyanpgupta) [reply src/pages/config/config-details.js:95]: @moshloop , before it was redirecting to previous page but as it is mentioned in the task to remove back button & add a link which redirect to home page and '/topology' is the route for the homepage
